pkg/actuators/machine/utils.go

```diff
@@ -180,6 +181,7 @@ func ProviderStatusFromRawExtension(rawExtension *runtime.RawExtension) (*machin
 	if rawExtension == nil {
 		return &machinev1.PowerVSMachineProviderStatus{}, nil
 	}
+	klog.Infof("ProviderStatusFromRawExtension: rawExtension = %+v", string(rawExtension.Raw))
```

Comment:
```suggestion
	klog.V(5).Infof("ProviderStatusFromRawExtension: rawExtension = %+v", string(rawExtension.Raw))
```

Reply:
@Karthik-K-N This change effectively hides the logging because all of the pods are started with the `--v=3` parameter.  This does not work for me.

Reply:
I changed it to `V(3)`.